src/pages/GetStarted/GetStartedPlaces.vue

```diff
@@ -214,9 +240,9 @@ export default {
 		clearFilter() {
 			this.filterTerm = '';
 		},
-		selectAllCountries() {
-			this.selectAll = !this.selectAll;
-			this.countryList.forEach(country => {
+		toggleAllCountries(state) {
+			this.selectAll = state;
+			this.filteredCountryList.forEach(country => {
```

Comment:
I believe this causes a bug in the select all functionality. If you filter for "Asia", hit "select all", then clear the Asia filter the asia countries are no longer selected (since you're now viewing this.countryList). If you type "as" you'll see the asia countries selected again (since you're viewing this.filteredCountryList where those items are selected).

One idea is to 
this.filteredCountryList.forEach(country => {
 

 
I believe this causes a bug in the select all functionality. If you filter for "Asia", hit "select all", then clear the Asia filter the asia countries are no longer selected (since you're now viewing this.countryList). If you type "as" you'll see the asia countries selected again (since you're viewing this.filteredCountryList where those items are selected).

Reply:
Good catch 👍